crates/generated_parser/src/ast_builder.rs

```diff
@@ -56,6 +56,10 @@ enum BindingKind {
 
     // BindingIdentifier is the name of ClassDeclaration.
     Class,
+
+    // BindingIdentifier is the name of Label.
```

Comment:
can you add some more comment that clarifies this is something unusual?
BindingIdentifier isn't Label per spec, but `LabelIdentifier`, and above sentence doesn't match.
so, this needs special comment that this is a hack that shares single vector for 2 purpose.

or maybe just add a reference to ` on_label_identifier` that has the comment.